### PR TITLE
BE-1264

### PR DESCRIPTION
**Pull request by KurtBrowder**
## Change description

> Description here

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request name and commit comments all contain JIRA user story

### CODE REVIEW

File: source/includes/client_sdk.md [hunk 1]
```diff
@@ -10,7 +10,13 @@
 </head>
 <body>
   <highline-link></highline-link>
-  <script src="https://link.highline.co/v1/highline-link.js"></script>
+
+  <!-- Sandbox -->
+  <script src="https://stg-link.highline.co/highline-link.js"></script>
+
+  <!-- Production -->
+  <!-- <script src="https://link.highline.co/v1/highline-link.js"></script> -->
```
Comment (recmend) [inline source/includes/client_sdk.md:18]: Why comment it?

File: source/includes/client_sdk.md [hunk 2]
```diff
@@ -34,10 +40,11 @@
...
 In order to ensure a simple and delightful experience for customers, we offer an integrated widget.
 
-Highline's widget is a front-end UI element that allows users to grant your application access to their work accounts, set up automated payments, and switch deposits directly from their paychecks.
+Highline's widget is a <a class="content-link" href="https://developer.mozilla.org/en-US/docs/Web/Web_Components" target="_blank">Web Component</a> that allows users to grant your application access to their work accounts, set up automated payments, and switch deposits directly from their paychecks.
 
-To use Highline-Link, pass in an object with the parameters described below to the  
-`highlineLink.connect()` function:
+To use Highline-Link, include the <span style='white-space:no-wrap;'>`<highline-link></highline-link>`</span> tag in your HTML document, then include the script tag for the environment you wish to target. 
+
+After these two tags are included on your page, connect to Highline-Link by passing in an object with the parameters described below to the `highlineLink.connect()` JavaScript function.  Once connected open the widget with `highlineLink.open()`.
```
Comment (recmend) [inline source/includes/client_sdk.md:47]: Do we need an example for highlineLink.open() e.g., onClickHandler or something?

**Review by recmend — COMMENTED** [review]
looks good, some questions.

File: source/includes/client_sdk.md (hunk 1 above)
Comment (KurtBrowder) [reply source/includes/client_sdk.md:18]: Just for clarity that they should have one or the other, not both.

File: source/includes/client_sdk.md (hunk 2 above)
Comment (KurtBrowder) [reply source/includes/client_sdk.md:47]: I don't think so, just wanted to go with the basics here.   Onclick isn't necessary to open it, though we could add some javascript guide to help inexperienced devs if needed.

Comment (recmend) [reply source/includes/client_sdk.md:47]: Might be worth adding something along the lines of
// `highlineLink.open()` would typically be user-initiated, such as a button click or tap

File: source/includes/introduction.md [hunk 3]
```diff
@@ -13,3 +13,42 @@ Highline's API is organized around REST. Our API has predictable, resource-orien
 ## Sandbox
 
 Our sandbox environment at `https://sandbox.highline.co` provides all the functionality of `https://api.highline.co` with test account credentials. The sandbox API credentials is also available in the product settings page at `https://app.highline.co/manage-product`.
+
```
Comment (recmend) [inline source/includes/introduction.md:16]: I'd add the production link url here as well. 

Highline-Link's sandbox url at `https://stg-link.highline.co/highline-link.js` provides all the functionality of `https://link.highline.co/v1/highline-link.js` with these credentials:

File: source/includes/introduction.md [hunk 4]
```diff
@@ -13,3 +13,42 @@ Highline's API is organized around REST. Our API has predictable, resource-orien
...
+Highline-Link's sandbox url is https://stg-highline-link.js and provides sandbox testing with these credentials:
```
Comment (recmend) [inline source/includes/introduction.md:17]: i think the https:// link here is incomplete.

**Review by recmend — APPROVED** [review]

**Pull request merged**